package org.example.task.exceptions;

public class UserNotFoundException extends RuntimeException{
    public UserNotFoundException(String s){
        super();
    }
}
